src/components/Navbar.tsx

```diff
@@ -132,42 +165,56 @@ export default function Navbar() {
             preview={false}
           />
         </Link>
-        <Menu
-          onClick={(e) => {
-            setselectedKeys([e.key]);
+
+        {/* Hamburger button for mobile */}
+        <Button
+          className="md:hidden"
+          type="text"
+          icon={<MenuOutlined />}
+          onClick={toggleDrawer}
+          style={{
```

Comment:
![image](https://github.com/user-attachments/assets/17f8940c-8e4c-4422-adbd-0d2a8b0947a1)

Not happy with the styling unfortunately. Would prefer it if it was larger and without the typical bordering etc.